contrib/packs/actions/install.meta.yaml

```diff
@@ -21,5 +21,5 @@
       default: "master"
     register:
       type: "string"
-      default: "actions"
+      default: "actions,aliases,sensors"
```

Comment:
This is misusing string parameter type (it should really be `array`), but I assume you have done it because you can't really specify arrays via chatops now, right?

If so, that's fine for now. As a future improvement, we could look into re-using CLI logic for specifying arrays and objects as strings, but this might require more work since aliases are not aware of the parameter types.